docs/public-networks/get-started/connect/sync-node.md

```diff
@@ -4,39 +4,132 @@ description: Full and archive node types
+
+!!! important
+
+    We recommend using snap sync over fast sync because snap sync can be faster by several days.
+    We recommend using snap sync with the [Bonsai](../../concepts/data-storage-formats.md#bonsai-tries)
+    data storage format for the fastest sync and lowest storage requirements.
+
+Enable snap sync using [`--sync-mode=X_SNAP`](../../reference/cli/options.md#sync-mode).
+You need Besu version 22.4.0 or later to use snap sync.
+
+Instead of downloading the [state trie](../../concepts/data-storage-formats.md) node by node, snap sync downloads as many leaves of the
+trie as possible, and reconstructs the trie locally.
+
+You can't switch from fast sync to snap sync.
+If your node is blocked in the middle of a fast sync, you can start over using snap sync instead by stopping the node,
+deleting the data directory, and starting over using `--sync-mode=X_SNAP`.
+
+You can restart Besu during a snap sync in case of hardware or software problems.
+The sync resumes from the last valid world state and continues to download blocks starting from the
+last downloaded block.
+
+See [how to read the Besu metrics charts](../../how-to/monitor/understand-metrics.md) when using snap sync.
+
+### Checkpoint synchronization
+
+!!! important
+
+    Checkpoint sync is an early access feature.
+
+Enable checkpoint sync using [`--sync-mode=X_CHECKPOINT`](../../reference/cli/options.md#sync-mode).
+You need Besu version 22.4.3 or later to use checkpoint sync.
+
+Checkpoint sync behaves like [snap sync](#snap-synchronization), but instead of syncing from the
+genesis block, it syncs from a specific checkpoint block configured in the [Besu genesis
+file](../../concepts/genesis-file.md).
+
+Ethereum Mainnet and the Goerli testnet configurations already define default checkpoints, so you
+don't have to add this yourself.
+
+For other networks, you can configure a checkpoint in the genesis file by specifying the block hash,
+number, and total difficulty as in the following example.
+
+!!! example "Checkpoint configuration example"
+
+    ```json
+    "checkpoint": {
+      "hash": "0x844d581cb00058d19f0584fb582fa2de208876ee56bbae27446a679baf4633f4",
+      "number": 14700000,
+      "totalDifficulty": "0xA2539264C62BF98CFC6"
+    }
+    ```
+
+!!! note
+
+    If using [Clique](../../../private-networks/how-to/configure/consensus/clique.md) consensus, the
+    checkpoint must be the beginning of an epoch.
+
+If you enable checkpoint sync without a checkpoint configuration in the genesis file, Besu snap
+syncs from the genesis block.
+
+You can restart Besu during a checkpoint sync in case of hardware or software problems.
+The sync resumes from the last valid world state and continues to download blocks starting from the
+last downloaded block.
 
 ### Fast synchronization
 
+!!! important
+
+    It might become impossible to sync Ethereum Mainnet using fast sync in the future.
+    Update Besu to a version that supports newer sync methods.
```

Comment:
It's a good callout, but I don't think FAST sync will happen again once the node is already in sync...so I think you only really need to update the config if you ever need to resync